### ISSUE TITLE
list disagrees with guess

### ISSUE THREAD

**Issue by packrat386** [Masterminds/glide#134]
`glide list` displays one set of dependencies but `glide guess` guesses a different list. Both lists contain dependencies that are directly imported as well as those that are dependencies of my dependencies, but `glide list` displays a longer list of dependencies. Why are these two commands using different criteria to determine if something is a dependency that needs to be managed?

**Comment by technosophos** [comment]
Yeah, `glide list` is very new, and uses an improved algorithm for determining dependencies. But it's still slow (and apparently can get caught in an infinite loop #136 ). So I am trying to improve its performance and then roll the new method back into `guess`.
